src/perform_step/low_order_rk_perform_step.jl

```diff
@@ -1029,3 +1029,111 @@ end
 
   #return nothing
 end
+
+function initialize!(integrator, cache::FRK65ConstantCache)
+  integrator.kshortsize = 9
+  integrator.k = typeof(integrator.k)(undef, integrator.kshortsize)
+  integrator.fsalfirst = integrator.f(integrator.uprev, integrator.p, integrator.t) # Pre-start fsal
+  integrator.destats.nf += 1
+
+  # Avoid undefined entries if k is an array of arrays
+  integrator.fsallast = zero(integrator.fsalfirst)
+  integrator.k[1] = integrator.fsalfirst
+  @inbounds for i in 2:integrator.kshortsize-1
+    integrator.k[i] = zero(integrator.fsalfirst)
+  end
+  integrator.k[integrator.kshortsize] = integrator.fsallast
+end
+
+@muladd function perform_step!(integrator, cache::FRK65ConstantCache, repeat_step=false)
+  @unpack t, dt, uprev, u, f, p = integrator
+  @unpack α21, α31, α41, α51, α61, α71, α81, α91, α32, α43, α53, α63, α73, α83, α54, α64, α74, α84, α94, α65, α75, α85, α95, α76, α86, α96, α87, α97, α98, β1, β4, β5, β6, β7, β8, β1tilde, β4tilde, β5tilde, β6tilde, β7tilde, β8tilde, β9tilde, c2, c3, c4, c5, c6, c7, c8, c9 = cache
+  alg = unwrap_alg(integrator, true)
+  omega = alg.omega
```

Comment:
looks like omega isn't used?

Reply:
Actually, I am testing for convergence of paper I mentioned in my earlier comment, which had constant variables. The paper for zero dissipation requires only to change b4 b5 b6 to a function of omega which I will implement after this just to be sure that original coefficients are leading to convergence or not.